src/Parser/Core/Modules/Others/DistanceMoved.js

```diff
@@ -66,13 +74,13 @@ class DistanceMoved extends Analyzer {
   }
 
   statistic() {
-    debug && console.log(`Total distance moved: ${this.totalDistanceMoved} yds`);
+    debug && console.log(`Time spent moving: ${this.timeSpentMoving / 1000} s, Total distance moved: ${this.totalDistanceMoved} yds`);
     return (
       <SmallStatisticBox
         icon={<Icon icon="spell_fire_burningspeed" />}
-        value={`≈${formatThousands(this.totalDistanceMoved)} yards`}
-        label="Distance moved"
-        tooltip={`≈${formatThousands(this.totalDistanceMoved / (this.owner.fightDuration / 1000) * 60)} yards per minute. Consider this when analyzing the fight, as some fights require more movement than others. Unnecessary movement can result in a DPS/HPS loss.`}
+        value={`≈${formatPercentage(this.timeSpentMoving / (this.owner.fightDuration))}%`}
+        label="Fight spent moving"
+        tooltip={`In ≈${formatThousands(this.timeSpentMoving / 1000)} seconds of movement you moved ≈${formatThousands(this.totalDistanceMoved)} yards. Consider this when analyzing the fight, as some fights require more movement than others. Unnecessary movement can result in a DPS/HPS loss.`}
```

Comment:
"Unnecessary movement can result in a DPS/HPS loss" is not really true for Tanks or Melee DPS, though I'm not sure if a reword is needed.

Reply:
This whole statistic calls for spec depended treatment in the future.  I would leave it as it is for now.

Reply:
I assume this also would count movement as a result of using abilities like Roll, Blink, Chi Torpedo, Charge, Heroic Leap, Disengage, Harpoon, etc ... when those really shouldnt count against the player ... imo

Reply:
@Sharrq this is why I created this from a simple distance moved in yards. Using any movement increase lowers % of time spend moving in fight (it counting for you).

Reply:
@Sharrq Well in this case it wouldn't "count against them" much, because when you use a movement ability you don't spend much time moving because you're moving very fast.

You're basically pointing out one of the advantages of switching from Distance Moved to Time Moving.

Reply:
Fair point, i didnt think about that.

Reply:
Basically my idea for Marksmanship (and to a degree BM because movement is irrelevant for them) is to implement their own spec-specific version of this and ensure that movement while X buff is up doesn't count as it allows movement while casting and then to alter the tooltip to showcase that information correctly. In that case it would only show the movement when it is actually a possible detriment (and show total movement in tooltip or something).

In any case I think this version of a movement module is significantly better than what we had before, as it's more lenient on the mobility spells.